Mathlib/Topology/ContinuousOn.lean

```diff
@@ -1495,6 +1495,17 @@ lemma ContinuousOn.union_continuousAt
   (fun h => ContinuousWithinAt.continuousAt (continuousWithinAt hs h) <| IsOpen.mem_nhds s_op h)
   (ht _)
 
+open Classical in
+/-- If a function is continuous on two closed sets, it is also continuous on their union. -/
+theorem ContinuousOn.union_isClosed {X Y : Type*} [TopologicalSpace X] [TopologicalSpace Y]
+    {s t : Set X} (hs : IsClosed s) (ht : IsClosed t) {f : X → Y} (hfs : ContinuousOn f s)
+    (hft : ContinuousOn f t) : ContinuousOn f (s ∪ t) := by
```

Comment:
I'm having a hard time thinking we don't have this, but I can't find it even with loogle. Can you make it a separate PR so as to lure people like Patrick into reviewing?